---
geometry_msgs/Vector3 pos1
geometry_msgs/Vector3 pos2
float64 yaw
 
